src/main/java/mate/academy/bookstore/dto/cartitem/UpdateShoppingCartItemRequestDto.java

```diff
@@ -0,0 +1,4 @@
+package mate.academy.bookstore.dto.cartitem;
+
+public record UpdateShoppingCartItemRequestDto(Integer quantity) {
```

Comment:
let's add validation

```suggestion
public record UpdateShoppingCartItemRequestDto(int quantity) {
```